unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx

```diff
@@ -101,12 +105,27 @@ export const EmailTemplatePreview = ({
           lockAddress,
           templateId,
           saveCustomContent.isSuccess,
+          customContent,
         ],
         queryFn: async () => {
-          const url = `${config.services.wedlocks.host}/preview/${templateId}`
+          const url = new URL(
+            `${config.services.wedlocks.host}/preview/${templateId}`
+          )
+
+          // parse markdown to HTML
+          const parsedContent = await unified()
```

Comment:
Does this sanitize the markdown as well?

Reply:
added thk!